website/docs/r/ssm_parameter.html.markdown

```diff
@@ -62,7 +62,6 @@ The following arguments are supported:
 * `value` - (Required) The value of the parameter.
 * `description` - (Optional) The description of the parameter.
 * `key_id` - (Optional) The KMS key id or arn for encrypting a SecureString.
-* `overwrite` - (Optional) Overwrite an existing parameter. If not specified, will default to `false` if the resource has not been created by terraform to avoid overwrite of existing resource and will default to `true` otherwise (terraform lifecycle rules should then be used to manage the update behavior).
```

Comment:
I'd consider leaving this in the docs for a bit with a deprecation notice and a note to use the import functionality.